ci-operator/config/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main.yaml

```diff
@@ -0,0 +1,48 @@
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+releases:
+  initial:
+    integration:
+      name: "4.15"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.15"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tests:
+- as: e2e
+  run_if_changed: ^.github/workflows/push.yaml$|^server|^operator|^ci
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-west-2
+    owner: rhtap
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.15"
```

Comment:
It looks like we only have clusterpools setup for 4.14 as the latest right now
https://github.com/openshift/release/tree/d83b0661d19b9f97c9a2a16118a51bd31fc8f6df/clusters/hive/pools/rhtap

I'm not sure if there is a reason not to have 4.15 besides maybe just the maintenance of making the switch. @psturc @flacatus ?

Reply:
Konflux staging clusters [were migrated to 4.14 very recently](https://redhat-internal.slack.com/archives/C02CTEB3MMF/p1712776304010559), so we did not need to test with 4.15 on PR basis yet. But now creating 4.15 cluster pool makes sense.

@flacatus on which version of ocp the RHTAP is tested (and is supported)?

Reply:
I had a chat with @dperaza4dustbit, sounds like we should be targeting 4.14 for now.  We can add 4.15 to the mix in a future PR once 4.15 clusters are available in the cluster pool (which will likely be a separate PR).